src/cozy/react/components/InstallationPage/components/InstallationStepMobilePart.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+
+import Stack from 'cozy-ui/transpiled/react/Stack'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { InstallNativeAppButton } from 'cozy/react/components/AvailablePlatforms'
+import mobileAppIcon from 'cozy/react/assets/mobile-app.svg'
+
+const InstallationStepMobilePart = ({ isNativePassInstalledOnDevice }) => {
```

Comment:
Maybe you can move the `useIsNativePassInstalledOnDevice` here so you wouldn't have to drill the `isNativePassInstalledOnDevice` on all the component tree (here seems to be the only place where you use it)

Reply:
It is because this way we don't to wait the intent in installation step mobile. If we wait and result is true, we can see it changing from "Install" to "Installed".